opencti-platform/opencti-graphql/src/http/httpPlatform.js

```diff
@@ -489,10 +489,11 @@ const createApp = async (app) => {
   });
 
   // Error handling
-  app.use((err, req, res) => {
+  app.use((err, req, res, next) => {
     const error = UnknownError('Http call interceptor fail', { cause: err, referer: req.headers.referer });
     logApp.error(error);
     res.status(500).send({ status: 'error', error: err.stack });
+    next(err);
```

Comment:
This part is a fix on top of https://github.com/OpenCTI-Platform/opencti/pull/8140